src/controllers/tasks/psc.statement.controller.ts

```diff
@@ -18,3 +19,17 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     return next(e);
   }
 };
+
+export const post = (req: Request, res: Response) => {
+  const pscButtonValue = req.body.pscStatementValue;
```

Comment:
could you wrap this in a try catch and return next(e) in the catch. This wil enable us to show the error page if any exceptions get thrown.

Reply:
i mean the contents of the post function